Item/tests/Item-specs.js

```diff
@@ -9,96 +9,97 @@ describe('Item Specs', () => {
+		const label = screen.getByText('Hello Item');
 
 		expect(label).toBeInTheDocument();
 	});
 
 	test('should support label with 0', () => {
 		const expected = '0';
 
-		const {getByText} = render(<ItemBase label={expected}>Hello Item</ItemBase>);
-		const label = getByText(expected);
+		render(<ItemBase label={expected}>Hello Item</ItemBase>);
+		const label = screen.getByText(expected);
 
 		expect(label).toBeInTheDocument();
 	});
 
 	test('should support adding text as a child when a label is also set', () => {
 		const expected = 'Hello Item';
 
-		const {getByText} = render(<ItemBase label="Example Label">{expected}</ItemBase>);
-		const actual = getByText(expected);
+		render(<ItemBase label="Example Label">{expected}</ItemBase>);
+		const actual = screen.getByText(expected);
 
 		expect(actual).toBeInTheDocument();
 	});
 
 	test('should support `slotBefore`', () => {
 		const expected = 'slot before';
 
-		const {getByText} = render(<ItemBase data-testid="item" slotBefore={expected}>Hello Item</ItemBase>);
-		const actual = getByText(expected);
+		render(<ItemBase data-testid="item" slotBefore={expected}>Hello Item</ItemBase>);
+		const actual = screen.getByText(expected);
 
 		expect(actual).toBeInTheDocument();
 	});
 
 	test('should support `slotAfter`', () => {
 		const expected = 'slot after';
 
-		const {getByText} = render(<ItemBase data-testid="item" slotAfter={expected}>Hello Item</ItemBase>);
-		const actual = getByText(expected);
+		render(<ItemBase data-testid="item" slotAfter={expected}>Hello Item</ItemBase>);
+		const actual = screen.getByText(expected);
 
 		expect(actual).toBeInTheDocument();
 	});
 
 	test('should support repositioning of the label', () => {
-		const {getByTestId} = render(<ItemBase data-testid="item" labelPosition="above" label="my label">Hello Item</ItemBase>);
+		render(<ItemBase data-testid="item" labelPosition="above" label="my label">Hello Item</ItemBase>);
+		const itemLabel = screen.getByTestId('item').children.item(1);
 
-		expect(getByTestId('item').children.item(1).className).toContain(css.labelAbove);
+		expect(itemLabel).toHaveClass(css.labelAbove);
 	});
 
 	test('should not include the selected class when not selected', () => {
-		const {getByTestId} = render(<ItemBase data-testid="item">Hello Item</ItemBase>);
+		render(<ItemBase data-testid="item">Hello Item</ItemBase>);
 
 		const expected = css.selected;
-		const actual = getByTestId('item').className;
+		const actual = screen.getByTestId('item');
 
-		expect(actual).not.toContain(expected);
+		expect(actual).not.toHaveClass(expected);
 	});
 
 	test('should add the selected class when given the selected prop', () => {
-		const {getByTestId} = render(<ItemBase data-testid="item" selected>Hello Item</ItemBase>);
+		render(<ItemBase data-testid="item" selected>Hello Item</ItemBase>);
 
 		const expected = css.selected;
-		const actual = getByTestId('item').className;
+		const actual = screen.getByTestId('item');
```

Comment:
Could you use getByText instead of getByTestId?

Reply:
In this case no. with getByText a child element is returned and the className 'selected' is assigned to the parent container of the component